lib/components/narrative/default/default-itinerary.js

```diff
@@ -192,3 +271,12 @@ export default class DefaultItinerary extends NarrativeItinerary {
     )
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    currency: state.otp.config.localization?.currency || 'USD',
```

Comment:
Should this not be derived from GTFS or the relevant GBFS (or other) providers? I'm not sure if there's a strong justification as to why we need this property.

Reply:
We are displaying a cost line item for car-only, walk-only and own-bike-only itineraries, even if no transit itinerary is found and the cost shown is $0.00. We need an "ambient" currency that we don't know if no transit itinerary is shown.
The alternative is to not show the cost item if `itinerary.fare` is not set. Thoughts?

Reply:
@landonreed says
> Ah, yea, good point, Binh. I'm fine with the ambient currency. That's helpful info that might be useful to throw into a comment though.

Comments regarding this added in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/9b7d311187b3fb919200faf427251bb58668cf0e.